### PR TITLE
Liberty: text editing and work on Harper's character

### PR DESCRIPTION
**Pull request by nemaara**
A quick edit of Liberty, fixing some dialogue, making Harper a girl and looking younger, editing some related (and unrelated) dialogue to go with it as well. Prep for a possible Harper reappearance in HttT.

### CODE REVIEW

File: data/campaigns/Liberty/images/portraits/harper.webp
Comment (cooljeanius) [inline data/campaigns/Liberty/images/portraits/harper.webp:1]: could we see this in another format for those of us unable to view webps?

Comment (knyghtmare) [reply data/campaigns/Liberty/images/portraits/harper.webp:1]: Go to Files -> View File -> "Show Raw"

Comment (cooljeanius) [reply data/campaigns/Liberty/images/portraits/harper.webp:1]: Screen Shot 2023-08-16 at 15 33 24

Comment (cooljeanius) [reply data/campaigns/Liberty/images/portraits/harper.webp:1]: ...ok, I managed to export a png version of it:
harper

Comment (CelticMinstrel) [reply data/campaigns/Liberty/images/portraits/harper.webp:1]: I can't put my finger on it, but something looks a bit off about the edit, I guess around the left shoulder…?

File: data/campaigns/Liberty/scenarios/07_Glory.cfg [hunk 1]
```diff
@@ -381,7 +381,7 @@ One score and eight years ago, in 70 YW, the King of Wesnoth and the Lord of Ele
         [/message]
         [message]
             type=Lancer
-            message= _ "At your order, my liege."
+            message= _ "At your order, my Liege."
```
Comment (CelticMinstrel) [inline data/campaigns/Liberty/scenarios/07_Glory.cfg:384]: I don't get why this is capitalized…?

File: data/campaigns/Liberty/images/portraits/harper.webp
Comment (nemaara) [reply data/campaigns/Liberty/images/portraits/harper.webp:1]: Which left, ours or Harper's?

File: data/campaigns/Liberty/scenarios/07_Glory.cfg (hunk 1 above)
Comment (nemaara) [reply data/campaigns/Liberty/scenarios/07_Glory.cfg:384]: Me neither, I'll fix it.
```suggestion
            message= _ "At your order, my liege."
```

File: data/campaigns/Liberty/images/portraits/harper.webp
Comment (CelticMinstrel) [reply data/campaigns/Liberty/images/portraits/harper.webp:1]: Harper's

Comment (nemaara) [reply data/campaigns/Liberty/images/portraits/harper.webp:1]: How about now (the original is still transparent)?

harper

**Pull request merged**